src/qt/bitcoingui.cpp

```diff
@@ -886,6 +893,19 @@ void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVer
     labelBlocksIcon->setToolTip(tooltip);
     progressBarLabel->setToolTip(tooltip);
     progressBar->setToolTip(tooltip);
+
+#ifdef Q_OS_WIN
+    if (m_taskbar_button->window() == nullptr) {
+        m_taskbar_button->setWindow(windowHandle());
+    }
+    QWinTaskbarProgress* taskbar_progress = m_taskbar_button->progress();
+    if (nVerificationProgress < 1.0) {
+        taskbar_progress->setValue(100.0 * nVerificationProgress + 0.5);
```

Comment:
Why `+ 0.5`? (add a comment)

Reply:
This is as same as here , I guess this is to round the value
https://github.com/bitcoin/bitcoin/blob/68f3c7eb080e461cfeac37f8db7034fe507241d0/src/qt/bitcoingui.cpp#L856

Reply:
Hm, okay.

Reply:
Use `qRound`?